CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ Main (unreleased)
 
 ### Enhancements
 
+- Helm: Selectively template ClusterIP. (@aglees)
```

Comment:
Can you move this to the CHANGELOG.md file in the Helm chart directory? We put Helm chart changes there.

Reply:
Hopefully that's all good now.